content/blog/virtuous-circles.md

```diff
@@ -0,0 +1,92 @@
+include csv files of input data, R code and tests, and the content of the paper itself. Rrtools offers a clear structure 
+in the configuration it expects from a compendium project - it's easy for the user to see what they need to fill in 
+without having to remember it, and some data can be auto-filled in from the user's profile. Rrtools also offers users a 
+host of nifty automation features to streamline the workflow, such as adding citations, dependencies and license, 
+auto-generating boilerplate parts of a readme file, and more technical tasks, like creating a github repo, setting up 
+continuous integration with Travis, and even creating a dockerfile which Travis can build and push to dockerhub.
+
+Next we heard from Maël Plaine about 
+the [reproducible articles](https://elifesciences.org/labs/7dbeb390/reproducible-document-stack-supporting-the-next-generation-research-article) project, 
+a collaboration between eLife and Substance, which considers the whole publication lifecycle and all of the users of a
+paper - reviewers and readers as well as authors. Like rrtools, the paper is augmented by data and code, and tools are 
+under development to make engaging with all these assets together seamless and transparent - for example the ability to 
+click on a plot in a paper, see the code which generated it, and potentially edit the code and instantly see the plot 
+update inline, all without leaving the browser. Research which can be engaged with in this way isn't just easily 
+reproducible, it's easily mutatable too. Tools like this may increase good practice by harnessing the value it adds, 
+increasing their usage and driving adoption of the technology as a standard. 
+
+These approaches don't just apply to publication. Our own [dettl](https://vimc.github.io/dettl/) package was developed 
+to help with data imports, in a context where messy, inconsistent data from disparate sources often needs to be imported 
+into our databases. The code to perform these imports was formerly written from scratch as and when required, with no 
+reference to the structure of previous imports, and tended to be monolithic, difficult to review and untested. Dettl ("disinfect 
+your workflow!") encourages a cleaner approach by providing methods for a staged approach, separated into Extract, 
+Transform and Load steps, each of which incorporate running a suite of tests, and where each step must succeed (pass all tests)
+ before the next can be attempted. Users of the package need put no effort into writing code with a good structure, or 
+ remembering to write a full test suite. 
+
+[Orderly](https://vimc.github.io/orderly/), another RESIDE project, provides an infrastructure for producing reports or 
+analyses by running code against specified inputs and collecting outputs. It shares this basic collection of code and 
+data with the publishing tools described above. However, Orderly is not focussed on publication, and offers additional 

```

Comment:
"focussed" should be "focused"

Reply:
Hornet's nest this one. I've got into trouble for 'focused' before. Apparently they're both in the dictionary, but you're right, one s is more common...
https://english.stackexchange.com/questions/4791/focussed-or-focused-rules-for-doubling-the-last-consonant-when-adding-ed